common/src/test/java/gov/cms/ab2d/common/model/CoverageDelta.java

```diff
@@ -0,0 +1,28 @@
+package gov.cms.ab2d.common.model;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.time.OffsetDateTime;
+
+@Entity
+@Data
+public class CoverageDelta {
```

Comment:
Why not extend/implement the base created/modified functionality?

Reply:
Never modified and not created through JPA.  This domain object is only a convenience for testing.